src/npc_class.h

```diff
@@ -33,6 +33,7 @@ class distribution
 
         distribution operator+( const distribution &other ) const;
         distribution operator*( const distribution &other ) const;
+        distribution& operator=( const distribution &other );
```

Comment:
The copy assignment operator should be automatically created by the compiler. Does your compiler complain without this?